components/gardener/extensions/extension-controllers/dns_external/chart/external-dns-management/templates/role.yaml

```diff
@@ -15,7 +15,7 @@ rules:
   resources:
   - configmaps
   resourceNames:
-  - dns-controller-manager
+  - {{ include "external-dns-management.fullname" . }}
```

Comment:
I changed that on purpose because the dns-controller will only work if that value is `dns-controller-manager`. Otherwise it is lacking the permissions to access the configmap from the leader election. 
Is there a specific reason why you reverted that change?

Reply:
(the current setup overwrites the `fullname` value, so it will actually break, unless something in the dns-controller-manager changed)